client-templates/github-enterprise/accept.json.sample

```diff
@@ -641,7 +641,7 @@
     {
       "//": "used to get repo's contributors list",
       "method": "GET",
-      "path": "/repos/:owner/:repo/contributors",
+      "path": "/repos/:owner/:repo/stats/contributors",
```

Comment:
are you sure regarding `stats`?
are we using this API https://developer.github.com/v3/repos/#list-contributors ?

Reply:
https://developer.github.com/v3/repos/statistics/#get-contributors-list-with-additions-deletions-and-commit-counts

Reply:
Yes, you right!

Reply:
👁 🦅